inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -127,11 +127,23 @@ def fuzzy_match(obj, eng):
 
     fuzzy_match_config = current_app.config['FUZZY_MATCH']
     matches = dedupe_list(match(obj.data, fuzzy_match_config))
-    record_ids = [el['_source']['control_number'] for el in matches]
+    record_ids = [_get_literature_brief(el['_source']) for el in matches]
     obj.extra_data.setdefault('matches', {})['fuzzy'] = record_ids[0:5]
     return bool(record_ids)
 
 
+def _get_literature_brief(literature):
+    brief = {
+        'control_number': literature['control_number'],
+        'title': get_value(literature, 'titles[0].title'),
+    }
+
+    abstract = get_value(literature, 'abstracts[0].value')
+    if abstract is not None:
```

Comment:
Maybe do just `if abstract:` is enough? Or are there empty string abstract values and we care about them?

Reply:
good point! abstracts are not required by the schema so in many cases they might be empty and is better not to send empty keys :)

Reply:
It is very unlikely the case but do you think there could be value in knowing if abstract is present but empty?

@drjova

Reply:
Since commit https://github.com/inspirehep/inspire-schemas/commit/470fff969b468fd68dff1e2c04ea89cfbaa75512 **every** list or string present in the schema if it's present is non-empty, so you don't have to deal with that case.